### ISSUE TITLE
Warmwasserzirkulation nicht umstellbar

### ISSUE THREAD

**Issue by RicoAeckerle** [TA2k/ioBroker.vaillant#33]
**Describe the bug**  
Ich würde gerne die Zirkulationspumpe über ioBroker steuern.
Bekomme diese aber nicht an. Auch ein ändern des Zeitplans ist nicht machbar.

**To Reproduce**  
set: vaillant.0.****.systemcontrol.dhw01.circulation.configuration.operationMode to: AUTO
Außerdem habe ich folgende Werte probiert, wobei AUTO drin stand und ich es nicht wieder erfolgreich setzten konnte:
1, 0, VLAUES01, VALUES02, MANUAL

**Screenshots & Logfiles**  
Failed to set: vaillant.0.****.systemcontrol.dhw01.circulation.configuration.operationMode to: AUTO

`<!doctype html><html lang=\"en\"><head><title>HTTP Status 404 – Not Found</title><style type=\"text/css\">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 404 – Not Found</h1><hr class=\"line\" /><p><b>Type</b> Status Report</p><p><b>Description</b> The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.</p><hr class=\"line\" /><h3>Apache Tomcat/mobilDIALOG-Live</h3></body></html>`

**Versions:**  
 - Adapter version: v0.1.2
 - JS-Controller version: 4.0.24
 - Node version: 16.19.0
 - Operating system: Docker / Synology

**Comment by kaese2002** [comment]
Geht Vaillantseitig nicht...Zirkulationspumpe ist immer der Warmwassergenerierung zugeordnet und kann nur über die Zeitprogramme gesteuert werden...also quasi in den Zeitprogrammen in denen die Warmwassergenerierung läuft, kann man die Zeitprogramme der ZP noch kastrieren. Einziger Workaround ist nen Minus auf X46/1 zu legen, also den Eingang für die Fernbedienung. Dann läuft die ZP extern angesteuert 5min. Ich hab das mit nem potentialfreien Sonoff gemacht, den ich über den iobroker steuere...

**Comment by menkej** [comment]
Cool, dass muss ich mal testen @kaese2002 - die Zeitprogramme laufen dann aber normal weiter, sprich: werden nicht ersetzt, oder?

**Comment by kaese2002** [comment]
Ja…der Eingang für die Fernbedienung hat keinen Einfluss auf die Zeitprogramme…

**Comment by menkej** [comment]
> Geht Vaillantseitig nicht...

Damit kann dieses Ticket ja eigentlich zu, oder?
